### PR TITLE
Decrease text size in mobile - Success stories page

### LINKED ISSUE #1233: Edit font size in mobile view in Success story page
**Describe the bug**
The font size in mobile view on the Success story page is too large

**To Reproduce**
1. Go to 'https://sefglobal.org/success_stories/'

**Expected behavior**
The text should be larger than the sub-text, but not too large in the mobile

**Screenshots**
image

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by dave-v0**
## Purpose

The purpose of this PR is to fix #1233 

## Goals

The goal is to decrease the size of the subparagraph on the success stories page

## Approach

Decreased text size in the CSS page
  
### Preview Link

https://pr-1245-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by dave-v0** [comment]
Can you take a look at this, please? @anjula-sack

**Comment by anjula-sack** [comment]
There are some conflicts @DilinaBandara. can you resolve them?

**Review by Piumal1999 — COMMENTED** [review]
@DilinaBandara Next time when you create a new branch, make sure the local main branch is up to date with the main branch of the sefglobal/sef-site repo. So you can avoid getting unwanted commits in the pull request.

```
git checkout main
git reset --hard upstream/main   (optional)
git pull upstream main
git checkout -b my-new-branch
// do the changes, commit and push
```

File: success_stories/styles.css [hunk 1]
```diff
@@ -161,4 +165,4 @@ hr {
 
 .mentor {
     margin-left: 0px;
-}
+}
```
Comment (Piumal1999) [inline success_stories/styles.css]:
```suggestion
}

```

Add EOF

File: success_stories/styles.css [hunk 2]
```diff
@@ -139,9 +142,10 @@ hr {
     margin-bottom: -1px;
 }
 
-.mentor-details .details{
+.mentor-details .details {
     line-height: 1.2rem;
-    margin-top: 4px;;
+    margin-top: 4px;
+    ;
```
Comment (Piumal1999) [inline success_stories/styles.css]: Remove the extra semicolon
```suggestion
    margin-top: 4px;
```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]

**Comment by dave-v0** [comment]
@Piumal1999 @anjula-sack Can you take a look at this, please?

**Pull request closed without merging**